src/score_table_models.py

```diff
@@ -226,7 +232,86 @@ class FileType(Base):
     updated_at = Column(DateTime)
     
     file_counts = relationship('ExptStoredFileCount', back_populates='file_type')
+
+class ExptArrayMetric(Base):
+    __tablename__ = EXPT_ARRAY_METRICS_TABLE
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    experiment_id = Column(Integer, ForeignKey('experiments.id'))
+    array_metric_type_id = Column(Integer, ForeignKey('array_metric_types.id'))
+    region_id = Column(Integer, ForeignKey('regions.id'))
+    value = Column(ARRAY(Float))
+    bias_correction = Column(ARRAY(Float))
+    assimilated = Column(Boolean)
+    time_valid = Column(DateTime)
+    forecast_hour = Column(Float)
+    ensemble_member = Column(Integer)
+    created_at = Column(DateTime, default=datetime.utcnow())
+
+    experiment = relationship('Experiment', back_populates='array_metrics')
+    array_metric_type = relationship('ArrayMetricType', back_populates='array_metrics')
+    region = relationship('Region', back_populates='array_metrics')
     
+class ArrayMetricType(Base):
+    __tablename__ = ARRAY_METRIC_TYPES_TABLE
+    __table_args__ = (
+        UniqueConstraint(
+            'name',
+            'measurement_type',
+            'measurement_units',
+            'stat_type',
+            'obs_platform',
+            'sat_meta_id',
+            name='unique_array_metric_type'
+        ),
+    )
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    sat_meta_id = Column(Integer, ForeignKey('sat_meta.id'), nullable=True)
+    obs_platform = Column(String(128))
+    name = Column(String(128), nullable=False)
+    long_name = Column(String(128))
+    measurement_type = Column(String(64), nullable=False)
+    measurement_units = Column(String(64))
+    stat_type = Column(String(64))
+    array_coord_labels = Column(ARRAY(String))
+    array_coord_units = Column(ARRAY(String))
+    array_index_values = Column(ARRAY(String))
+    array_dimensions = Column(ARRAY(String))
+    description = Column(JSONB(astext_type=sa.Text()), nullable=True)
+    created_at = Column(DateTime, nullable=False)
+    updated_at = Column(DateTime)
+
+    array_metrics = relationship('ExptArrayMetric', back_populates='array_metric_type')
+    sat_meta = relationship('SatMeta', back_populates='array_metric_type')
+
+class SatMeta(Base):
+    __tablename__ = SAT_META_TABLE
```

Comment:
This is an excellent point to bring up here. I will bring this to the whole team and see what our use cases look like on this, but it would be good to get this decided before completing this PR so the table is a stable form.

Reply:
Per the offline team discussion, the current table design will work the best for our use cases so we'll keep this structure.